code/controllers/subsystem/explosions.dm

```diff
@@ -373,8 +373,10 @@ SUBSYSTEM_DEF(explosions)
 				SSexplosions.medturf += T
 			if(EXPLODE_LIGHT)
 				SSexplosions.lowturf += T
-
-
+		//SKYRAT EDIT ADDITION
+		for(var/obj/machinery/light/iterating_light in T)
```

Comment:
I assume it isn't intentional that this toggles low-power/dying lights to full and then off?
https://gyazo.com/45edfff25390ca159fca4cc8f08df05f

Reply:
oh, i actually think this is maybe to be flickering on/off but is flickering on/lowpower..

Reply:
It was fixed, server not updated